### PR TITLE
fix(testharness): fix typo and explanation

### PR DESCRIPTION
**Pull request by marcoscaceres**
Small typo and clarification... the current message was little bit confusing (for me), so hopefully this helps

### CODE REVIEW

**Review by Ms2ger — APPROVED** [review]
Most likely the CI issues are not your fault

**Comment by past** [comment]
The test failure seems unrelated, I will admin merge this.

File: resources/testharness.js [hunk 1]
```diff
@@ -2174,7 +2174,7 @@
             func = funcOrConstructor;
             description = descriptionOrFunc;
             assert(maybeDescription === undefined,
-                   "Too many args pased to no-constructor version of assert_throws_dom");
+                   "Too many args passed to no-constructor version of promise_rejects_dom, or accidentally explicitly passed undefined");
```
Comment (zcorpan) [inline resources/testharness.js:2177]: `assert_throws_dom`

Comment (zcorpan) [reply resources/testharness.js:2177]: Oh, this was merged. :) I can open a new PR

Comment (zcorpan) [reply resources/testharness.js:2177]: #46566

**Pull request merged**